tests/config/test-utils.tsx

```diff
@@ -0,0 +1,30 @@
+import { render, RenderResult } from "@testing-library/react";
+import { AppProps } from "next/app";
+import React from "react";
+import { ThemeProvider } from "styled-components";
+
+import { Layout } from "../../src/components/Layout";
+import { GlobalStyle } from "../../src/design-system/globals/globalStyle";
+import { lightTheme } from "../../src/design-system/theme/lightTheme";
+
+const AllTheProviders: React.FC<JSX.Element> = ({ children }) => {
+  return (
+    <ThemeProvider theme={lightTheme}>
+      <GlobalStyle />
+      <Layout>{children}</Layout>
+    </ThemeProvider>
+  );
+};
```

Comment:
Could that be done with the global App component?

Reply:
Do you mean testing the app component rather than the index page ?

Reply:
No, I mean not copy/pasting the providers but using the App component instead to avoid duplication and forgetting to add things there

Reply:
I'm not sure I understand what you mean.

Reply:
I mean, I understand the issue, but not what you have in mind as a solution.